akka-actor-typed/src/main/scala/akka/actor/typed/internal/StashBufferImpl.scala

```diff
@@ -88,26 +89,24 @@ import akka.util.ConstantFun
 
   override def forEach(f: Consumer[T]): Unit = foreach(f.accept(_))
 
-  override def unstashAll(ctx: scaladsl.ActorContext[T], behavior: Behavior[T]): Behavior[T] =
-    unstash(ctx, behavior, size, ConstantFun.scalaIdentityFunction[T])
+  override def unstashAll(behavior: Behavior[T]): Behavior[T] =
+    unstash(behavior, size, ConstantFun.scalaIdentityFunction[T])
 
-  override def unstashAll(ctx: javadsl.ActorContext[T], behavior: Behavior[T]): Behavior[T] =
-    unstashAll(ctx.asScala, behavior)
-
-  override def unstash(ctx: scaladsl.ActorContext[T], behavior: Behavior[T],
-                       numberOfMessages: Int, wrap: T ⇒ T): Behavior[T] = {
+  override def unstash(
+    behavior:         Behavior[T],
+    numberOfMessages: Int, wrap: T ⇒ T): Behavior[T] = {
```

Comment:
I realize now this causes a conflict on Scala 2.12 where scala Functions are SAMs, will have to revert and keep that ctx parameter or some other trick.